app/components/publications/publication/proofs/proof-upload-modal.hbs

```diff
@@ -29,6 +29,14 @@
                   <Auk::Label>{{t "file-type"}}</Auk::Label>
                   <Utils::Documents::FileTypePill @extension={{this.file.extension}} />
                 </div>
+                {{#if (eq @source "proof")}}
+                  <div class="auk-form-group {{this.validators.receivedAtDate.errorClass}}">
+                    <Auk::Label for="proofUploadReceivedAtDate">{{t "received-at"}}</Auk::Label>
+                    <Auk::Datepicker id="proofUploadReceivedAtDate"
+                      @date={{this.receivedAtDate}}
+                      @onChange={{this.setReceivedAtDate}} />
+                  </div>
```

Comment:
een ontvangen drukproef krijgt een received date, maar een gecorrigeerde drukproef niet?
Ben niet helemaal mee met de flow hiervan, kan dus correct zijn.

Reply:
Dit klopt. Een ontvangen drukproef krijgt OVRB van een externe partij. Ze willen dan de datum waarop ze dat document ontvangen hebben registeren (want dat is niet noodzakelijk de datum waarop ze het document uploaden).
Het gecorrigeerde drukproef document is een document dat door OVRB zelf opgesteld wordt om opnieuw naar het BS te sturen. Aangezien ze dat document zelf gemaakt hebben en uploaden moeten ze hier geen ontvangsdatum voor registreren.